docs/getting-started/custom-theme.mdx

```diff
@@ -1,6 +1,8 @@
 # Custom Theme
```

Comment:
Is this used under "Getting started? Can't see it in the navigation

Reply:
Not at the moment, I'll focus on custom addons and themes later for the docs.